lib/action_cable/channel/streams.rb

```diff
@@ -12,7 +12,7 @@ module Channel
     #     def follow(data)
     #       stream_from "comments_for_#{data['recording_id']}"
```

Comment:
I originally thought the above sample could be expressed with `stream_for` but I see now that it can't. So yes, a separate example would be great :+1: